pkg/ddevapp/ddevapp.go

```diff
@@ -1257,6 +1257,8 @@ Fix with 'ddev config global --required-docker-compose-version="" --use-docker-c
 	}
 
 	// Build extra layers on web and db images if necessary
+	output.UserOut.Printf("Building project containers... This can take some time.")
+	buildDurationStart := util.ElapsedDuration(time.Now())
 	progress := "quiet"
 	if globalconfig.DdevVerbose {
```

Comment:
The next thing that would be hugely useful here is a good way to do the verbose build output.

Reply:
Not sure I understand this part, and would this be something you wanna do as part of this PR?